src/js/plugins/plugin.text_selection.js

```diff
@@ -0,0 +1,204 @@
+//@ts-check
```

Comment:
Can you check what the file size of the built plugin file is? We might need to strip back somethings to make it smaller 😬  (I.e. async/await)

Reply:
plugin.text_selection.js is 61KB and plugin.text_selection.js.map is 258KB. There are other plugins of comparable size such as tts, search and url.

Reply:
👍 Few! That seems good :) (once we switch to webpack 5, this will drop _waaaay_ down, too!)